### PR TITLE
add required mapbox credit

### LINKED ISSUE #504: Mapbox attribution
[Map in footer needs attribution](https://www.mapbox.com/help/attribution/). 

screen shot 2017-02-13 at 11 14 49

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Fixes #504 as per [their recommendation](https://www.mapbox.com/help/attribution/)

### Large
mapbox-d

### Medium
mapbox-t

## For interface components
- [x] Browser testing - Have you checked the component in our supported browsers
- [x] No javascript - Have you checked the component with javascript disabled
- [ ] A11y testing - Have you run pa11y against the component and fixed/verified all errors, warnings and notices?

FYI @Norvard

### CODE REVIEW

File: client/scss/components/_find_us.scss [hunk 1]
```diff
@@ -52,12 +55,51 @@
 }
 
 .find-us__map {
-  max-width: 100%;
+  width: 100%;
```
Comment (gestchild) [inline client/scss/components/_find_us.scss]: Will need to adjust the srcset and sizes attributes, as the image can now appear larger on smaller screens

**Comment by gestchild** [comment]
Looks good. The only issue may be that there are now 2 links to google maps directions and they have different link text. May cause accessibility issues:

[H2: Combining adjacent image and text links for the same resource](https://www.w3.org/TR/WCAG20-TECHS/H2.html)

[3.2.4 Consistent Identification](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#consistent-behavior-consistent-functionality)

File: client/scss/components/_find_us.scss (hunk 1 above)
Comment (jamesgorrie) [inline client/scss/components/_find_us.scss]: You might need to run me through why we chose these sizes in the first place...

**Comment by jamesgorrie** [comment]
@gestchild I wasn't aware of that spec. I have pushed the links down a little so they no longer overlap the map.

I have also made them 10px - do you see that being a problem? I notices gMaps do (which doesn't mean anything really).

mapbox

**Comment by gestchild** [comment]
@jamesgorrie Hmm good question

[3.2 Touch Target Size and Spacing](https://www.w3.org/TR/mobile-accessibility-mapping/#touch-target-size-and-spacing) recommends that the size of touch targets should be a minimum of 9mm by 9mm, with inactive space around them.

[Size Tap Targets Appropriately](https://developers.google.com/speed/docs/insights/SizeTapTargetsAppropriately) says infrequently used links could be smaller the 7mm, but with 5mm (32 CSS pixels) of space around them.

Not sure how that works with design @Norvard ?

**Comment by Norvard** [comment]
Yea... those are good rules but this seems like such an edge case use. How many people will actually want to click on those links? I would guess very little... so maybe its ok we leave them small here. My 2 cents, not based on any hard evidence but just gut feeling

**Pull request merged**